src/Matrix.test.js

```diff
@@ -85,4 +85,24 @@ describe("Matrix tests", () => {
     }
   });
 
+  test("multiplication of values",async() => {
```

Comment:
Move this to calculate.test.js

Reply:
addMatrix is in both, will do but should also remove addMatrix from Matrix.test